### PR TITLE
test(atenlib): explicitly convert ints to int64

### LINKED ISSUE #309: Broken CI: test_output_match_arange_cpu_float32
Example job: https://github.com/microsoft/onnx-script/actions/runs/3893754501/jobs/6646787599 
```
SUBFAIL onnxscript/test/function_libs/torch_aten/ops_correctness_test.py::TestOutputConsistencyCPU::test_output_match_arange_cpu_float32 - AssertionError: The values for attribute 'dtype' do not match: torch.int32 != torch.int64.
SUBFAIL onnxscript/test/function_libs/torch_aten/ops_correctness_test.py::TestOutputConsistencyCPU::test_output_match_arange_start_step_cpu_float32 - AssertionError: The values for attribute 'dtype' do not match: torch.int32 != torch.int64.
```

### PR DESCRIPTION
**Pull request by justinchuby**
This should fix #309. Green ci 1/2.

Also set codecov to be informational only so CI status is not affected

### CODE REVIEW

**Comment by justinchuby** [comment]
Apparently it doesn't fix the problem

**Review by fatcat-z — APPROVED** [review]
LGTM, thanks!

File: onnxscript/test/function_libs/torch_aten/ops_correctness_test.py [hunk 1]
```diff
@@ -171,6 +171,18 @@ def _amax_amin_input_wrangler(
     return args, kwargs
 
 
+def _arange_input_wrangler(
+    args: list[Any], kwargs: dict[str, Any]
+) -> tuple[list[Any], dict[str, Any]]:
+    new_args = []
+    for arg in args:
+        if isinstance(arg, int):
+            # Explicitly convert to int64 because ints are 32-bit on Windows
```
Comment (fatcat-z) [inline onnxscript/test/function_libs/torch_aten/ops_correctness_test.py:180]: nit:

...because int type is 32-bit on Windows.

Comment (justinchuby) [reply onnxscript/test/function_libs/torch_aten/ops_correctness_test.py]: Done

File: onnxscript/test/function_libs/torch_aten/ops_correctness_test.py [hunk 2]
```diff
@@ -171,6 +171,18 @@ def _amax_amin_input_wrangler(
     return args, kwargs
 
 
+def _arange_input_wrangler(
```
Comment (BowenBao) [inline onnxscript/test/function_libs/torch_aten/ops_correctness_test.py:174]: Why move from previous fix to adding 'wrangler', given this is a general issue not specific to arange?
Are you planning a more fundamental fix in onnx-script?

**Pull request merged**